### DISCUSSION TITLE
Dropdown inside modal.

### DISCUSSION THREAD

**Discussion by Cesar-Ornelas** [saadeghi/daisyui#2720] [General]
I used the dropdown with an input to create a calendar and it works great. Only issue I have noticed is when the dropdown is inside a modal it will expand only inside the modal. Does anyone have an idea of how-to popup the content similar to how the select component works. 

This shows how both act inside a modal:

[https://play.tailwindcss.com/CvNzv7HxsW](https://play.tailwindcss.com/CvNzv7HxsW)

**Comment by saadeghi** [comment]
You can add `overflow-y-visible` to `modal-box` but make sure your list can fit the screen, because currently, it's a long list:  
https://play.tailwindcss.com/vVlfJwts1A

Generally I wouldn't suggest putting interactive parts inside each other. Even if it works perfectly, It's not good for UX that clicking A opens B and clicking something in B opens C, etc. It's too many moving parts and it would be hard to use for the users.

Let me know if you have a question.
